src/hclib-runtime.c

```diff
@@ -354,14 +355,15 @@ static inline void check_out_finish(finish_t *finish) {
         // hc_atomic_dec returns true when finish->counter goes to zero
         if (hc_atomic_dec(&(finish->counter))) {
 #if HCLIB_LITECTX_STRATEGY
```

Comment:
What says everyone to making it so HCLIB_LITECTX_STRATEGY is the only option now? I doubt the other code path is well-tested at this point anyway.